src/Controller/PasswordController.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Admin;
+use App\FormData\PasswordData;
+use App\Form\ResetPasswordType;
+use Symfony\Component\Form\FormError;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\IsGranted;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\Security\Core\Encoder\UserPasswordEncoderInterface;
+
+class PasswordController extends AbstractController
+{
+    /**
+     * @Route("/resetPassword", name="reset-password")
+     * @IsGranted("ROLE_ADMIN")
+     */
+    public function edit(Request $request, UserPasswordEncoderInterface $passwordEncoder): Response
+    {
+        $result = $this->getDoctrine()->getManager();
```

Comment:
Strange name for the EntityManager... and you definitely should use the controller parameters to get Services in controllers